notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4446,6 +4446,55 @@ fear of commitment [Clementine accused Joel of this shortcoming.]
 love triangle [Harold, his wife, and the receptionist Mary, but Mary latter having wiped her memories of it.]
 
 
+movie: Immortal (2004)
+======================
+
+:: Title
+Immortal
+
+:: Description
+Immortal (French: Immortel, ad vitam) is a 2004 English language French
+live-action and animated science fiction film co-written and directed by Enki
+Bilal and starring Linda Hardy, Thomas Kretschmann and Charlotte Rampling. It
+is loosely based upon Bilal's comic book La Foire aux immortels (The Carnival
+of Immortals).
+
+Synopsis: The Ancient Egyptian god Horus, who is evidently a space alien,
+seeks to procreate with a human female, after being stripped of his
+immortality by his fellow gods.
+
+:: Date
+2004-03-24
+
+:: References
+https://en.wikipedia.org/wiki/Immortal_(2004_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+coping with mortality [The god/alien Horus was determined to mate with a human female after being stripped of his immortality. Having offspring was presumably his way of coping with the reality that he would eventually die.]
+speculative genetic engineering [The story is set in a a future where genetically altered humans lived side by side with unaltered men and women. Much was made of the heroine Jill being some sort of genetically engineered person.]
+Ancient Egyptian mythology [The story focused on the Ancient Egyptian god Horus, and how he lost his immortality.]
+extraterrestrial being [A revelation of the film is that the ancient Egyptian gods were really aliens.]
+the god is an alien [A revelation of the film is that the ancient Egyptian gods were really advanced technology wielding aliens.]
+aliens impregnating human females [The god Horus, who was actually an alien, made a number of clandestine attempts to impregnate Jill.]
+romantic love [Jill and Nikopol ended up falling in love.]
+what if I had multiple personalities [The film explores Nikopol having two selves: His ordinary self and the rapist self he became whenever Horus inhabited his body.]
```

Comment:
🤔

Reply:
I think the problem is we don't have a theme to the effect of "what if my body became inhabited by another entity". We do, however, have these the related themes **spirit possession**, demonic possession**, **what if a loved one became possessed by another being**, and **evil hand**.